docs/vtex-io/Store Framework Apps/vtex-gift-list/vtex-list-listmigration.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "List Migration"
+slug: "vtex-list-listmigration"
+excerpt: "vtex.list@3.6.2"
+hidden: false
+createdAt: "2023-01-16T00:32:17.866Z"
+updatedAt: "2023-01-16T00:32:17.866Z"
+---
+
+VTEX has a native feature for list management, that can be verified on the [following document](https://help.vtex.com/en/subcategory/list-types--6aeeWirCBUwwCmeIWOgIWs). However, this feature works only for CMS stores, making it impossible to use for IO and Faststore stores.
+
+**Gift List app** was made to fill this gap, allowing all types of stores to work with lists. Because of that, a common demand is related to the migration of lists, from the native feature to the new app feature.
+
+Is important to take a look at the following details to execute this process:
+
+- The access page for lists will change. On the native feature, the access occurs by the `/giftlist` path. On the new feature, the access occurs by the home page subaccount where the app was installed;
+- Both features work with a giftcard system to transfer purchase values to the guests for the list owner. Credits that a client receives in the native feature are valid for the app feature;
+- Is necessary to migrate pieces of information related to gift lists, as list owners and items linked with each list;
+- The credits transaction history isn't kept on the native feature. Because of that, isn't possible to migrate this data for the app feature.
```

Comment:
```suggestion
- The credit transaction history isn't kept on the native feature. Because of that, it isn't possible to migrate this data for the app feature.
```